HMNFD_Design.md

```diff
@@ -0,0 +1,274 @@
+# Cray State Change Notification Service (hmnfd)
+
+The Shasta State Change Notification (SCN) fanout service provides the ability
+to notify subscribers of component hardware state changes and other changes
+made to and by the Hardware State Manager.  _hmnfd_ is a companion service
+to the Hardware State Manager in that it does the heavy lifting of 
+distributing state changes and managing subscriptions.
+
+This service uses a RESTful interface to provide the following functions:
+
+* Subscription for component state changes:
+  * Hardware state (On, Off, Ready, etc.)
```

Comment:
is Ready, really a hardware state?

Reply:
Technically not but it is lumped into the HW state by HSM.  The Logical State field is not where Ready
is ever held.